src/test/java/seedu/address/logic/commands/ListPatientCommandTest.java

```diff
@@ -4,39 +4,39 @@
 import static seedu.address.logic.commands.CommandTestUtil.showPersonAtIndex;
 import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
 import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalPersons.getTypicalAppointmentSchedule;
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import seedu.address.model.Model;
 import seedu.address.model.ModelManager;
 import seedu.address.model.UserPrefs;
-import seedu.address.testutil.AppointmentScheduleBuilder;
 
 /**
- * Contains integration tests (interaction with the Model) and unit tests for ListCommand.
+ * Contains integration tests (interaction with the Model) and unit tests for ListPatientCommand.
  */
-public class ListCommandTest {
+public class ListPatientCommandTest {
 
     private Model model;
     private Model expectedModel;
 
     @BeforeEach
     public void setUp() {
-        model = new ModelManager(new AppointmentScheduleBuilder().build(), getTypicalAddressBook(),
+        model = new ModelManager(getTypicalAppointmentSchedule(), getTypicalAddressBook(),
                 new UserPrefs());
-        expectedModel = new ModelManager(new AppointmentScheduleBuilder().build(), model.getAddressBook(),
+        expectedModel = new ModelManager(getTypicalAppointmentSchedule(), model.getAddressBook(),
```

Comment:
This one should be getAppointmentSchedule instead

Reply:
don't think so, logic is we don't want any patient commands to affect appointment schedule, so we make a new typicalAppointmentSchedule here